java/codebuild/sourceCredential/README.md

```diff
@@ -0,0 +1,18 @@
+# Welcome to your CDK Java project!
+
+It is a Maven-based project, so you can open this directory with any Maven-compatible Java IDE, and you should be able to build and run tests from your IDE.
+
+You should explore the contents of this template. It demonstrates a CDK app to create a codebuild source credential cloud formation resource.
```

Comment:
Once again, I need a little more context here. What is a source credential and why should I care?
See previous README comment for suggestions to resolve.

Reply:
Updated